_posts/2024-02-07-#28950.md

```diff
@@ -8,16 +8,50 @@ authors: [instagibbs]
 components: ["rpc"]
 host: ismaelsadeeq
 status: upcoming
-commit:
+commit: bf6fbea9794db5e8d0adce8ff6182574808cea54
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
+- The [`submitpackage RPC`](https://bitcoincore.org/en/doc/26.0.0/rpc/rawtransactions/submitpackage/) RPC method takes `package` as a required argument. The package must consist of a child with its parents, and none of the parents may depend on one another. The package will be validated according to consensus and mempool policy rules. If all transactions pass, they will be accepted to mempool.
+
+- Package transactions submitted to the `submitpackage` RPC successfully are accepted to the node's mempool, but broadcasted to peers individually because [Package relay](https://bitcoinops.org/en/topics/package-relay/) is currently a work in progress. Successful submission does not mean the transactions will propagate throughout the network whenever the transactions need to be relayed as a package.
```

Comment:
```suggestion
- Package transactions submitted to the `submitpackage` RPC successfully are accepted to the node's mempool, but broadcasted to peers individually because [Package relay](https://bitcoinops.org/en/topics/package-relay/) is currently a work in progress. Successful submission does not mean the transactions will propagate.
```

Reply:
Also not sure how this is relevant to the PR. I think it's more relevant to mention the CPFP use case (which can lead to overshooting on fees) instead. That might make this text feel less out of place.

Reply:
Removed